quiver/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM phusion/baseimage:0.9.19
```

Comment:
We started using baseimage for its protection against zombie processes - important for Chrome on Docker. Since your `ENTRYPOINT` doesn't prefix the binary name with `/sbin/my_init`, we're not using that protection. My guess is Node.js doesn't leave zombie processes around on Docker so we could go with plain old `ubuntu:xenial` here.